registry/flutter_reactive_ble.test

```diff
@@ -0,0 +1,8 @@
+contact=miguel.catarino.medeiros@signify.com
+contact=remon.helmond@signify.com
+fetch=git -c core.longPaths=true clone https://github.com/PhilipsHue/flutter_reactive_ble.git tests
+fetch=git -c core.longPaths=true -C tests checkout 741b46ae40228f4de5b7bd0d40bc014c4472a1db
+update=.
+test=flutter packages get
```

Comment:
this shouldn't be necessary, doesn't `flutter analyze` take care of it?

Reply:
I didn't have it initially, but then I got some errors as mentioned in https://github.com/flutter/tests/pull/30#issuecomment-602730131.

After adding this line the errors went away.